native/src/main/resources/META-INF/services/io.ballerina.projects.buildtools.CodeGeneratorTool

```diff
@@ -0,0 +1 @@
+io.ballerina.consolidate.CodeGenerator
```

Comment:
```suggestion
io.ballerina.consolidate.CodeGenerator

```

Reply:
Fixed in https://github.com/ballerina-platform/consolidate-packages-tool/commit/bd473123358220cca26188349581184775fe47ca